eva-server/src/test/java/uk/ac/ebi/eva/server/ws/ga4gh/GA4GHBeaconWSServerTest.java

```diff
@@ -55,7 +55,8 @@
 
     @Before
     public void setUp() throws Exception {
-        VariantWithSamplesAndAnnotation variant = new VariantWithSamplesAndAnnotation("1", 1000, 1005, "reference", "alternate");
+        VariantWithSamplesAndAnnotation variant = new VariantWithSamplesAndAnnotation("1", 1000, 1005, "reference",
+                                                                                      "alternate", "rs1");
```

Comment:
Same comment that in GeneWSServerTest.java

Reply:
for the GA4GH web services the services from `VariantWithSamplesAndAnnotationsService` are called but do not return the results from it but a `GASearchVariantsResponse` or `GA4GHBeaconResponse`. So the `createdDate field is not returned for this web serices.